outscale/data_source_outscale_vm.go

```diff
@@ -351,28 +347,24 @@ func getOApiVMAttributesSchema() map[string]*schema.Schema {
 		},
 		"image_id": {
 			Type:     schema.TypeString,
-			ForceNew: true,
-			Optional: true,
 			Computed: true,
 		},
 		"is_source_dest_checked": {
 			Type:     schema.TypeBool,
-			Optional: true,
 			Computed: true,
 		},
 		"keypair_name": {
 			Type:     schema.TypeString,
-			Optional: true,
 			Computed: true,
 		},
 		"security_group_ids": {
 			Type:     schema.TypeSet,
-			Optional: true,
+			Computed: true,
```

Comment:
is it okay, to change that from optional to computed ?

Reply:
Yes, that is the data_source, we cannot set  parameters directly.